### ISSUE TITLE
telegraf 1.25.2 is not code-signed for windows amd-x64

### ISSUE THREAD

**Issue by shawn3231** [influxdata/telegraf#12732]
### Relevant telegraf.conf

```toml
I pulled the latest release of telegraf x64 for windows (v1.25.2) and noticed that it is not code-signed. Because of other security tools, unsigned applications are automatically blocked. If the application is code signed, we can add that to a list of trusted sources.
```

### Logs from Telegraf

```text
no logs
```

### System info

1.25.2

### Steps to reproduce

1. Open the properties of telegraf.exe
2. Their should a digital signature tab
3. Pull up the information of the influx certificate

### Expected behavior

See the certificate being used

### Actual behavior

Digital Signature tab is missing

### Additional info

https://github.com/influxdata/telegraf/issues/6423
https://github.com/influxdata/telegraf/pull/8816

**Comment by powersj** [comment]
hmm when the CircleCI breach was announced at the beginning of the year we rotated all our signing certs. Looking at the [nightly builds](https://app.circleci.com/pipelines/github/influxdata/telegraf/15140/workflows/8d7846fd-c765-4cfb-9479-5573891038db/jobs/238547) it looks like it reports signing was successful and valid. Looking at the [v1.25.2 tag](https://app.circleci.com/pipelines/github/influxdata/telegraf/14993/workflows/97eabd59-9b43-4c35-9f2d-4848f02c9058/jobs/236514), while it passes it is not working and throwing:

> The specified network password is not correct.

When rotating these, the first cert I got did not have a password on it, but later did. The release branch has the modified signing script. I have reverted the signign script to the original value via https://github.com/influxdata/telegraf/commit/7b64dcebb6871dfa9b029ea7173483537a59de2a

The v1.25.3 release schedule for Monday should then have proper signatures.
